src/components/CardItem.js

```diff
@@ -1,24 +1,30 @@
 import React, { useState } from "react";
-import styled from "styled-components";
+import styled from "styled-components/macro";
+
+import Heart from "react-animated-heart";
 
 function CardItem(props) {
```

Comment:
I would encourage you to destructure the props here:
```suggestion
function CardItem({label, src, header, text, /*and so on*/}) {
```